### ISSUE TITLE
Disabled parking showing as time-limited

### ISSUE THREAD

**Issue by riiga** [zlant/parking-lanes#133]
Time-limited disabled parking using the new schema shows as time-limited, but probably should show as disabled parking only.

[Map example](https://zlant.github.io/parking-lanes/#19/58.41212/15.62470) ([OSM way](https://www.openstreetmap.org/way/746403752))

Relevant tags:
```
parking:right:access = no
parking:right:disabled = designated
parking:right:fee = no
parking:right:maxstay = 3 hours
```

**Comment by davidpnewton** [comment]
Yes. There is no obvious way to get disabled parking to show up using the new tagging scheme.

A loading bay is easy as simply setting the restriction to loading_only will do it. Disabled parking? Not nearly so obvious.
